pkg/flowkit/util/utilities.go

```diff
@@ -187,3 +190,57 @@ func AddToGitIgnore(filename string, loader ReaderWriter) error {
 		filePermissions,
 	)
 }
+
+type jsonConfig struct {
+	Metrics bool `json:"metrics"`
+}
+
+// AddToConfig adds a new line to user's config dir stating the user's preference for command tracking
+func AddToConfig(loader ReaderWriter, metricsEnabled bool) error {
+	homeConfigDir, err := os.UserConfigDir()
```

Comment:
using user config dir, in my machine it is $HOME/Library/Application Support, so that it works automatically for windows and other os